test_conformance/math_brute_force/binary_double.cpp

```diff
@@ -317,7 +317,16 @@ cl_int Test(cl_uint job_id, cl_uint thread_id, void *data)
         // Fill the result buffer with garbage, so that old results don't carry
         // over
         uint32_t pattern = 0xffffdead;
-        memset_pattern4(out[j], &pattern, buffer_size);
+
+        if ((error = clEnqueueFillBuffer(tinfo->tQueue, tinfo->outBuf[j],
+                                         &pattern, sizeof(pattern), 0,
+                                         buffer_size, 0, NULL, NULL)))
```

Comment:
I think the preceding call to clEnqueueMapBuffer and the call to clEnqueueUnmapMemObject immediately following should be removed after adding the calls to clEnqueueFillBuffer because the intialization is being done entirely on the device now, correct?